src/Monthly/MonthlyBody.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { ReactNode, useContext } from 'react';
+import { useMonthlyCalendar } from './MonthlyCalendar';
+import { daysInWeek } from '../shared';
+import { format, getDay, isSameDay } from 'date-fns';
+
+const MonthlyBodyContext = React.createContext({} as any);
+type BodyState<DayData> = {
+  day: Date;
+  events: DayData[];
+};
+
+export function useMonthlyBody<DayData>() {
+  return useContext<BodyState<DayData>>(MonthlyBodyContext);
+}
+
+type OmittedDaysProps = {
+  days: Date[];
+  omitDays?: number[];
+};
+
+const handleOmittedDays = ({ days, omitDays }: OmittedDaysProps) => {
```

Comment:
How do you feel about exporting this ?

Reply:
sure, if anyone may need it, then it seems like a good idea